Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/deltaonboarding/ui/DeltaonboardingFragment.kt

```diff
@@ -60,6 +62,14 @@ class DeltaonboardingFragment : Fragment(R.layout.fragment_test_deltaonboarding)
         binding.switchVaccinationOnboarding.setOnCheckedChangeListener { _, value ->
             viewModel.setVaccinationRegistrationOnboardingDone(value)
         }
+
+        binding.switchNotificationsDeltaOnboarding.setOnCheckedChangeListener { _, value ->
+            viewModel.setNotificationsOnboardingDone(value)
+        }
+
+        binding.switchAnalyticsDeltaOnboarding.setOnCheckedChangeListener { _, value ->
+            viewModel.setAnalyticsOnboardingDone(value)
+        }
     }
 
```

Comment:
very tiny remark: Please capitalize the first letter in "Onboarding" in the title to be consistent with other files

Reply:
Good point: done ✅